controllers/comments.controllers.js

```diff
@@ -16,4 +17,18 @@ function getCommentsPerArticleID(req, res, next) {
     });
 }
 
-module.exports = { getCommentsPerArticleID };
+function postComment (req, res, next) {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+  return selectArticleById(article_id).then(() => {
```

Comment:
This would be a perfect time for a Promise.all or look at the advanced error handling notes.